### PR TITLE
turn off logging every single line of a shell script

### PR DESCRIPTION
**Pull request by penguhyang**
fix issue: #816

### CODE REVIEW

File: xCAT-server/share/xcat/install/scripts/chroot.sles [hunk 1]
```diff
@@ -12,6 +13,11 @@ cat /tmp/pre-install.log >> /mnt/var/log/xcat/xcat.log
 
 echo "Running AutoYaST Chroot-Installation script..."
 
+export XCATDEBUGMODE="#TABLEBLANKOKAY:site:key=xcatdebugmode:value#"
+if [ "$XCATDEBUGMODE" = "on" ]; then
+    set -x
+fi
+
 #!/bin/sh
```
Comment (immarvin) [inline xCAT-server/share/xcat/install/scripts/chroot.sles]: the line should be in the beginning of the script

Comment (penguhyang) [inline xCAT-server/share/xcat/install/scripts/chroot.sles]: fixed

File: xCAT-server/share/xcat/install/scripts/pre.sles [hunk 2]
```diff
@@ -2,12 +2,18 @@
...
 <![CDATA[
 {
 echo "Running AutoYaST Pre-Installation script..."
 
+export XCATDEBUGMODE="#TABLEBLANKOKAY:site:key=xcatdebugmode:value#"
+if [ "$XCATDEBUGMODE" = "on" ]; then
+    set -x
+fi
+
 #!/bin/sh
```
Comment (immarvin) [inline xCAT-server/share/xcat/install/scripts/pre.sles]: hi, "#!/bin/sh" should be always the 1st line of the script file

File: xCAT-server/share/xcat/install/scripts/post.sles.common [hunk 3]
```diff
@@ -5,6 +5,10 @@ export XCATDEBUGMODE="#TABLEBLANKOKAY:site:key=xcatdebugmode:value#"
 export MASTER_IP="#ENV:MASTER_IP#"
 #INCLUDE:#ENV:XCATROOT#/share/xcat/install/scripts/scriptlib#
 
+if [ "$XCATDEBUGMODE" = "1" ] || [ "$XCATDEBUGMODE" = "2" ]; then
```
Comment (immarvin) [inline xCAT-server/share/xcat/install/scripts/post.sles.common]: these lines should br placed under " #!/bin/sh"

File: xCAT-server/share/xcat/install/scripts/pre.sles [hunk 4]
```diff
@@ -2,12 +2,18 @@
...
 <![CDATA[
 {
 echo "Running AutoYaST Pre-Installation script..."
 
+export XCATDEBUGMODE="#TABLEBLANKOKAY:site:key=xcatdebugmode:value#"
+if [ "$XCATDEBUGMODE" = "1" ] || [ "$XCATDEBUGMODE" = "2" ]; then
+    set -x
+fi
+
 #!/bin/sh
```
Comment (immarvin) [inline xCAT-server/share/xcat/install/scripts/pre.sles]: `` #!/bin/sh `` should  be placed at the beginning of script

File: xCAT-server/share/xcat/install/scripts/post.sles.common (hunk 3 above)
Comment (penguhyang) [inline xCAT-server/share/xcat/install/scripts/post.sles.common]: Yes, these lines already under " #!/bin/sh"

File: xCAT-server/share/xcat/install/scripts/pre.sles (hunk 4 above)
Comment (penguhyang) [inline xCAT-server/share/xcat/install/scripts/pre.sles]: fixed

**Pull request merged**